src/Fields/Link.php

```diff
@@ -21,7 +21,9 @@ public static function resolve(...$parameters): ?Link
 
     public static function resolveMany($items): Collection
     {
-        return resolve(LinkResolver::class)->resolveMany($items);
+        return collect($items)->map(function($item) {
```

Comment:
I think we either store the logic on the LinkResolver as use this method as an alias, or get rid of the LinkResolver class in favour of putting it on here?

Reply:
Yeah, worth considering the role of these classes and are they structured in the best way for their purpose.

Reply:
The `resolveMany()` method isn't used in any of our implementations so I'm going to remove it. The main issue with resolving many items is the title override, so we'd need to find an expressive way to determine this. At the moment it's easy enough to map from a collection:

```php
    public function getRelatedAttribute($items): IlluminateCollection
    {
        Query::make()->cache()->findMany(
            collect($items)->pluck('related_item.id')->filter()->toArray()
        );

        return collect($items)
            ->map(function($item) {
                return Link::resolve(
                    data_get($item, 'related_item'),
                    data_get($item, 'related_title')
                );
            })
            ->filter();
    }
```